Lists/Lab/GaussTrick/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace GaussTrick
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            List<int> numbers = Console.ReadLine()
+                .Split(' ')
+                .Select(int.Parse)
+                .ToList();
+
+            GaussTrick(numbers);
+
```

Comment:
This should not be part of the Main method. The method name should contain a verb, indicating what the method is actually doing